src/main/java/DaveTheBrave.java

```diff
@@ -0,0 +1,369 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Paths;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+import java.time.*;
+
+class Task {
+    private String type; // "T" = To-Do, "D" = Deadline, "E" = Events
+    private String task;
```

Comment:
this variable name is the same as the class name. It is better to have a different meaningful name.